docs/reference/indices/index-templates.asciidoc

```diff
@@ -76,19 +93,35 @@ PUT _component_template/component_template1
   }
 }
 
-PUT _component_template/other_component_template
+PUT _component_template/runtime_component_template
 {
   "template": {
     "mappings": {
+      "runtime": { <1>
+        "day_of_week": {
+          "type": "keyword",
+          "script": {
+            "source": "emit(doc['@timestamp'].value.dayOfWeekEnum.getDisplayName(TextStyle.FULL, Locale.ROOT))"
+          }
+        }
+      },
       "properties": {
-        "ip_address": {
-          "type": "ip"
+        "@timestamp": {
```

Comment:
because it's a component template, timestamp does not necessarily need to be defined in the same template?

Reply:
@javanna, can you please elaborate? I don't understand the connection between the timestamp and component template.

Reply:
I was referring to the fact that the timestamp is already defined in the other component template, and you replaced the previous ip_address in this component template with the timestamp. I thought that is not necessary....but am I missing something?

Reply:
It's also a very common pattern for users to have separate component templates for mappings and settings. For example, our built-in component templates do this: https://www.elastic.co/guide/en/elasticsearch/reference/master/index-management-settings.html#stack-templates-enabled

Reply:
No, no, you're correct. I was completely overlooking the timestamp in the other component template. I'll remove this one 👍

Reply:
NVM. I misread. Please ignore my drive-by comment.